docs/pages/setup/reference/moderated-sessions.mdx

```diff
@@ -0,0 +1,115 @@
+Here is a collection of details on configuring Moderated Sessions.
+
+## RBAC
+
+Moderated Sessions makes use of RBAC policies to allow for fine grained control over
+who can join a session and who are required to be present to start one.
+
+The system is based around **require policies** and allow policies. Require policies define a set of conditions
+that must be a met for a session to start/run.
+
+**Allow policies** are used to define what sessions a user can join
+and under what conditions they may join a session.
+
+### Session kinds
+
+Require and allow policies have to specify which sessions they apply to. Valid options are `ssh` and `k8s`.
+
+### Filters
+
+A filter expression or predicate is a string statement used to define logic based on a set of input variables.
+These follow common programming language syntax and allow for more detailed control.
+`user.name == "adam" || contains(user.traits, "view")` is a simple example of a filter expression
+that evaluates to true if the user is Adam or if the user has the trait `view`.
+
+Require policies can specify which users they consider as valid with a filter expression. They work like
+other filters used within Teleport such with access requests. The filter context has a `user` object defined
+which the set fields `roles`, `traits` and `name`.
+
+### Backwards compatibility with Server Access
+
+Previously, Server Access did not include controls over which users can join a session.
+To work around this, RBAC rules are ignored for users that only have V4 roles (`version: v4` in the role specification).
+New roles are created as V5. V4 roles are upgraded when they are modified in the UI.
+If a user has any attached V5 roles, the new RBAC access checks will be enforced.
+
+### Example require policy
+
+The policy below specifies that a given role
+must have a minimum of two users with the role `auditor` and the mode `moderator` present in the session
+to start it. The policy applies to SSH and Kubernetes sessions only.
+
+When a user with this require policy starts a session, it will be pending
+until the policy is fulfilled.
+
+```yaml
+kind: role
+metadata:
+  name: prod-access
+spec:
+  allow:
+    require_session_join:
+      - name: Auditor oversight
+        filter: 'contains(user.roles, "auditor")'
+        kinds: ['k8s', 'ssh']
+        modes: ['moderator']
+```
+
+### Example allow policy
+
+The following allow policy attaches to the role `auditor` and allows one to join
+SSH and Kubernetes sessions started by a user with the role `prod_access` as a moderator or observer.
+
+```yaml
+kind: role
+metadata:
+  name: auditor
+spec:
+  allow:
+    join_sessions:
+      - name: Auditor oversight
+        roles : ['prod-access']
+        kinds: ['k8s', 'ssh']
+        modes: ['moderator', 'observer']
+```
+
+## MFA-based presence
+
+When `mfa_required` is set to `true` via role or cluster settings, Teleport enforces
```

Comment:
- If MFA-based presence is another RBAC config setting, should we include it in the previous section (i.e., as an H3 heading) rather than as a new H2-level section?
- I think it would be helpful to add examples of what `mfa_required` looks like in role or cluster settings. This would make it clear where in a config spec we would expect to find this field (e.g., directly under `spec` or under another field).

Reply:
I got the MFA setting name wrong. We've got a page on per session MFA which is now linked.